Source/DiabloUI/diabloui.cpp

```diff
@@ -589,7 +589,7 @@ void LoadHeros()
 
 void LoadUiGFX()
 {
-	ArtLogo = LoadPcxSpriteList("ui_art\\hf_logo2", /*numFrames=*/16, /*transparentColor=*/0);
+	ArtLogo = LoadPcxSpriteList("ui_art\\hf_logo2", /*numFrames=*/16, /*transparentColor=*/0, nullptr, false);

```

Comment:
`nullptr, false`, what's that for?

Reply:
In the previous PR i changed it to always try to load the hellfire logo and then check if it's there, but I overlooked that `LoadPcxSpriteList()` will output a warning unless you specify `logerrors = false`.